### ISSUE TITLE
[Feature] Duplicate of duplicate: put new image aside old one

### ISSUE THREAD

**Issue by Germano0** [darktable-org/darktable#7457]
**TL;DR:** I have a list of duplicates of an image, and I want to duplicate a duplicate of the photo. The new duplicate should appear next to the old one, darktable instead, puts the new one at the end of the list of duplicates.
**Long version:**
Let's say I have an image `A`. In lighttable when I click on "duplicate" I get the new image (let's call it `A_1`) next to `A`, and you can see a yellow border around them.
Now imagine I want to test some different retouch solutions, so I duplicate A various times and I apply various retouch solutions to all the variants. I will get a list of photos with hypothetical names `A, A_1, A_2, A_3, A_4`.
Let's imagine among all variants I like `A_2` the most and I want to test some small changes, so I select `A_2`, and click on duplicate button. I will get a duplicate `A_2_2`. The new duplicate should appear next to the old one, darktable instead, puts the new one at the end of the list of duplicates, giving me a `A, A_1, A_2, A_3, A_4, A_2_2`.
Before filling this ticket, I was doing very small changes on some duplicates, and when I duplicated a duplicate, I could not understand where the new duplicate was, because all the duplicates were very similar each other

**Comment by parafin** [comment]
The problem is that duplicates are numbered flat. so there is no A_2_2 possible, only A_5. Only solution is to re-number other duplicates, which I don't think is a good idea. So this doesn't look like easily solvable.

**Comment by TurboGit** [comment]
Maybe using the "custom sort" we can have this ? But would be more a workaround than a full feature.

**Comment by Germano0** [comment]
writing a message to avoid autoclose
